client/package.json

```diff
@@ -1,4 +1,6 @@
 {
+  "//*":"NOTE: THIS REPOSITORY CONTAINS TWO PACKAGE.JSON FILES",
+  "//**":"FIND THE SERVER-SIDE PACKAGE.JSON FILE IN THE PARENT DIRECTORY",
```

Comment:
Apparently this is a kooky way to add "comments" to json?

Reply:
haha yeah this is the one thing I _really_ hate about JSON!  If I could go back in time, I'd force the guy who invented it to support the doggone things.